src/emmetHelper.ts

```diff
@@ -383,14 +383,31 @@ function getFilters(text: string, pos: number): { pos: number, filter: string }
 /**
  * Extracts abbreviation from the given position in the given document
  */
-export function extractAbbreviation(document: TextDocument, position: Position, lookAhead: boolean = true): { abbreviation: string, abbreviationRange: Range, filter: string } {
+export function extractAbbreviation(document: TextDocument, position: Position, extractOptions?: boolean | { lookAhead?: boolean, syntax?: string }): { abbreviation: string, abbreviationRange: Range, filter: string } {
 	const currentLine = getCurrentLine(document, position);
 	const currentLineTillPosition = currentLine.substr(0, position.character);
 	const { pos, filter } = getFilters(currentLineTillPosition, position.character);
 	const lengthOccupiedByFilter = filter ? filter.length + 1 : 0;
 
 	try {
-		const result = extract(currentLine, pos, lookAhead);
+		let options: boolean | { lookAhead: boolean, syntax: string };
+		if (!extractOptions) {
```

Comment:
When `false` is passed as `extractOptions`, this will end up setting lookAhead to true. So swap the `if`s. First check if type is boolean and then check for `!extractOptions`